share/location/osm.py

```diff
@@ -147,3 +160,60 @@ def osm_resolve_name(name, country=DEFAULT_COUNTRY_CODE):
     db.session.commit()
 
     return locations[0]
+
+
+def osm_resolve_coordinates(latitude, longitude):
+  
+    q = {
+        'format': 'json',
+        'lat': latitude,
+        'lon': longitude,
+    }
```

Comment:
Purely a matter a preference here but using a value such as `q` kind of robs the variable of its contextual meaning, from face value its a bit difficult to tell what `q` is. Seems like its a query parameter so I'd call it that